src/libcore/array/mod.rs

```diff
@@ -1,5 +1,5 @@
 //! Implementations of things like `Eq` for fixed-length arrays
-//! up to a certain length. Eventually we should able to generalize
+//! up to a certain length. Eventually we should be able to generalize
```

Comment:
```suggestion
//! up to a certain length. Eventually, we should be able to generalize
```
Also feel like this would read slightly better but I can't find consistent documentation for it (it looks like grammarly would [prefer it](https://www.quora.com/Is-there-always-a-comma-after-the-word-eventually-unless-it-s-at-the-end-of-a-sentence-Or-is-Grammarly-being-overly-controlling) but the internet isn't so sure 🤷‍♂